xlators/cluster/dht/src/dht-common.c

```diff
@@ -10964,12 +10964,12 @@ dht_notify(xlator_t *this, int event, void *data, ...)
                     goto unlock;
                 if ((cmd == GF_DEFRAG_CMD_STATUS) ||
                     (cmd == GF_DEFRAG_CMD_DETACH_STATUS))
-                    gf_defrag_status_get(conf, output);
+                    gf_defrag_status_get(conf, output, false);
```

Comment:
... of course, also `s/false/_gf_false/`

Reply:
Done